js/index.js

```diff
@@ -1,42 +1,175 @@
+    'about-h4': 'About',
+    'about-content': 'About content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.',
+    'middle-img-src': 'img/mid-page-accent.jpg',
+    'services-h4': 'Services',
+    'services-content': 'Services content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.',
+    'product-h4': 'Product',
+    'product-content': 'Product content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.',
+    'vision-h4': 'Vision',
+    'vision-content': 'Vision content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.',
   },
-  "contact": {
-    "contact-h4" : "Contact",
-    "address" : "123 Way 456 Street Somewhere, USA",
-    "phone" : "1 (888) 888-8888",
-    "email" : "sales@greatidea.io",
+  contact: {
+    'contact-h4': 'Contact',
+    address: '123 Way 456 Street Somewhere, USA',
+    phone: '1 (888) 888-8888',
+    email: 'sales@greatidea.io',
   },
-  "footer": {
-    "copyright" : "Copyright Great Idea! 2018"
+  footer: {
+    copyright: 'Copyright Great Idea! 2018',
   },
 };
 
 // Example: Update the img src for the logo
-let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+const logo = document.getElementById('logo-img');
+logo.setAttribute('src', siteContent.nav['img-src']);
+
+const headerImage = document.getElementById('cta-img');
+headerImage.setAttribute('src', siteContent.cta['img-src']);
+
+const middleImage = document.getElementById('middle-img');
+middleImage.setAttribute('src', siteContent['main-content']['middle-img-src']);
+
+const btn = document.querySelector('button');
+const btn2 = siteContent.cta.button;
+btn.textContent = btn2;
+
+const heading = document.querySelector('h1');
+const heading2 = siteContent.cta.h1;
+heading.textContent = heading2;
+
+const navs = document.querySelectorAll('a');
+const nav1 = siteContent.nav['nav-item-1'];
+
+navs[0].textContent = nav1;
+
+const nav2 = siteContent.nav['nav-item-2'];
+navs[1].textContent = nav2;
+
+const nav3 = siteContent.nav['nav-item-3'];
+navs[2].textContent = nav3;
+
+const nav4 = siteContent.nav['nav-item-4'];
+navs[3].textContent = nav4;
+
+const nav5 = siteContent.nav['nav-item-5'];
+navs[4].textContent = nav5;
+
+const nav6 = siteContent.nav['nav-item-6'];
+navs[5].textContent = nav6;
+
+
+const topContentFeaturesHeading = document.querySelector('.top-content .text-content:nth-child(odd) h4');
```

Comment:
Here you can put `nth-child(1)` to avoid unpredictable behaviour in the future otherwise things could mess up if you add a new paragraph in the same `text-content` container for example.